src/config.js

```diff
@@ -33,8 +33,16 @@ module.exports = {
     hostname: process.env.ANALYTICS_HOSTNAME || "",
   },
 
+  postgres: {
+    host : process.env.POSTGRES_HOST,
+    user : process.env.POSTGRES_USER,
+    password : process.env.POSTGRES_PASSWORD,
+    database : process.env.POSTGRES_DATABASE || "analytics-reporter",
+  },
+
   static: {
-    path: '../analytics.usa.gov/'
+    path: '../analytics.usa.gov/',
   },
 
+  timezone: process.env.ANALYTICS_TIMEZONE || "US/Eastern",
```

Comment:
One thought I had typing about this: Is there TZ info in the response we get from Google?

I'll prob let this lie and open an issue later to investigate better ways to handle the TZ for the GA data.